doc/conf.py

```diff
@@ -207,7 +207,16 @@
 html_static_path = ["_static"]
 
 # Custom CSS for the site
-html_css_files = ["custom_styles.css"]
+html_css_files = [
+    "driver.min.css",
+    "custom_styles.css",
+    ]
```

Comment:
```suggestion
]
```

Reply:
Yeah, I guess, to be consistent with what the rest of the conf.py uses